src/multiselect/multiselect.component.tsx

```diff
@@ -40,7 +40,8 @@ function OutsideAlerter(props) {
 }
 
 export class Multiselect extends React.Component<IMultiselectProps, any> {
-  static defaultProps: { customArrow: any; className: string; options: never[]; disablePreSelectedValues: boolean; selectedValues: never[]; isObject: boolean; displayValue: string; showCheckbox: boolean; selectionLimit: number; placeholder: string; groupBy: string; style: {}; emptyRecordMsg: string; onSelect: () => void; onRemove: () => void;onKeyPressFn: ()=>void; closeIcon: string; singleSelect: boolean; caseSensitiveSearch: boolean; id: string; closeOnSelect: boolean; avoidHighlightFirstOption: boolean; hidePlaceholder: boolean; showArrow: boolean; keepSearchTerm: boolean; };
+  static defaultProps: { customArrow: any; class
```

Comment:
Is thr any reason changed className into class?

Reply:
Sorry, its a typo.
Added another commit to reset it back.

Can you correct it if I'm doing wrong? Make sure we can see name as attribute in select component.